src/main/java/net/pms/dlna/DLNAMediaAudio.java

```diff
@@ -509,40 +509,63 @@ public String getAudioCodec() {
 	@Override
 	public String toString() {
 		StringBuilder result = new StringBuilder();
-		result.append("id: ");
-		result.append(getId());
-		result.append(", lang: ");
-		result.append(getLang());
+		if (!getLang().equals("und")) {
+			result.append("Id: ").append(getId());
+			result.append(", Language Code: ").append(getLang());
+		}
 
 		if (isNotBlank(getAudioTrackTitleFromMetadata())) {
-			result.append(", audio track title from metadata: ");
-			result.append(getAudioTrackTitleFromMetadata());
+			if (result.length() > 0) {
+				result.append(", ");
+			}
+			result.append("Audio Track Title From Metadata: ").append(getAudioTrackTitleFromMetadata());
 		}
 
-		result.append(", audio codec: ");
-		result.append(getAudioCodec());
-		result.append(", sample frequency:");
-		result.append(getSampleFrequency());
-
-		if (getAudioProperties() != null && getAudioProperties().getNumberOfChannels() != 0) {
-			result.append(", number of channels: ");
-			result.append(getAudioProperties().getNumberOfChannels());
+		if (result.length() > 0) {
+			result.append(", ");
+		}
+		result.append("Audio Codec: ").append(getAudioCodec());
+		if (getSampleFrequency() != null) {
+			result.append(", Sample Frequency: ").append(getSampleFrequency());
+		}
+		if (getBitRate() != 0) {
+			result.append(", Bitrate: ").append(getBitRate());
+		}
+		if (getBitsperSample() != 0 && getBitsperSample() != 16 ) {
```

Comment:
@Sami32 I don't think you quite understand ```toString()```. It's not only used in the log that you think of, it's used automatically to "print" a class instance. It's a standard Java method that's used to generate a "string representation of an object". It will be used under many different circumstances, for example when using a debugger. The person logging at this output might be looking for something completely different than what you're usually looking for. 

If something is due to a bug in MediaInfo is irrelevant, this is the primary way to view an instance as a string, which means that you need to get all the relevant information out. How are we for example supposed to know that MediaInfo (or something else, a bug in our code) sets this to 0 when 0 is hidden? It could be the reason why something doesn't work, and hiding it might make that much more difficult to detect. Ideally I don't like hiding default values either, but because these classes are "over crowded" with information leaving out default/blank/null values makes some sense.